### ISSUE TITLE
API Editor: Edit server info (base URL, etc)

### ISSUE THREAD

**Issue by EricWittmann** [Apicurio/apicurio-studio#118]
There is some global server information that should be editable, such as scheme(s), base URL, etc.  This information should be editable.

**Comment by SamDelamarter** [comment]
I worked on this during innovation days -- looking to have a PR out in a week or so depending on my availability.

**Comment by EricWittmann** [comment]
Awesome thanks!  I'm actively working on support for OAI 3.0.0 (on a branch) and so am looking forward to a gruesome merge with your stuff. :)

**Comment by pblommendaal** [comment]
+1 Would be nice to have instead of adding it via code.

**Comment by EricWittmann** [comment]
Oh boy this is old, eh?  I wonder if @SamDelamarter has any old code that could be contributed via a PR (even if not completed) so I can finish it up?

**Comment by EricWittmann** [comment]
Note: this feature is OpenAPI 2.0 only.

**Comment by pblommendaal** [comment]
Yes, unfortunately we still need to use the OpenAPI 2.0 specification.

**Comment by pblommendaal** [comment]
Yes, unfortunately we still need to use the OpenAPI 2.0 specification.

**Comment by EricWittmann** [comment]
Yeah that's no problem - I was just noting it in case I'm not the one fixing this issue. :)

**Comment by lucamaf** [comment]
even when adding 'schemes' using the Source tab this is removed automatically, is this the expected behaviour?
